src/main/java/org/assertj/core/internal/Iterables.java

```diff
@@ -917,6 +919,18 @@ public void assertContainsExactly(AssertionInfo info, Iterable<?> actual, Object
                            shouldContainExactly(actual, values, diff.missing, diff.unexpected, comparisonStrategy));
   }
 
+  public void containsAnyOf(AssertionInfo info, Iterable<?> actual, Object[] values) {
+    checkIsNotNull(values);
+    assertNotNull(info, actual);
```

Comment:
I believe these checks can be replaced by `commonCheckThatIterableAssertionSucceeds` that would also handle the empty case.